src/main/java/edu/greenblitz/robotName/subsystems/shooter/Funnel/FunnelConstants.java

```diff
@@ -0,0 +1,40 @@
+package edu.greenblitz.robotName.subsystems.shooter.Funnel;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkMaxLimitSwitch;
+import edu.greenblitz.robotName.Robot;
+import edu.greenblitz.robotName.RobotConstants;
+import edu.greenblitz.robotName.utils.motors.GBSparkMax;
+
+
+public class FunnelConstants {
+
+	public static final double ROLL_POWER = 1;
```

Comment:
add EJECT_POWER instrad of saying -ROLL_POWER in Funnel class

Reply:
i dont see why. its just adding lines for fun

Reply:
> i dont see why. its just adding lines for fun

you will probably want to eject at a higher speed than rolling